Makefile

```diff
@@ -0,0 +1,31 @@
+ROOT_DIR							?= /scratch/msc22f9/mempool
```

Comment:
We shouldn't add a makefile to the top directory for this. Ff we use this to invoke `riscv_tests`, we should move it to an appropriate Subdirectory and add it to CI. How to handle this best should be clearer once #381 is merged.

Also, please see the CI annotations for general lint fixes; here, you need a license header.